compute/k8s-services/vars.tf

```diff
@@ -930,6 +930,12 @@ variable "velero_bucket_name" {
   description = "The name of the S3 bucket that contains the Velero backup"
 }
 
+variable "velero_cron_schedule" {
```

Comment:
Did you forget to push? I don't see velero_cron_schedule being used anywhere

Reply:
https://github.com/dfds/infrastructure-modules/pull/839/files#diff-9dcb2b34c331c83c7b15b8785fc38080f22e765d3119f854acc7848e7f669485R764

Reply:
My eyes, my eyes :)